src/pages/AlumniPage.jsx

```diff
@@ -1,90 +1,179 @@
+                            <h1 className="text-black text-3xl font-[700] mt-[3%] ">Alumni Scholarships for Tomorrow's Leaders</h1>
+                            <hr className="w-16 h-1  bg-[#FFA523] border-0 rounded mt-[5%]"/>
+                            <p className="text-xs text-black mt-[5%] leading-[22px]">FUSION is proud to offer alumni-sponsored scholarships, a testament to the strong support and dedication 
+                                of our alumni network. These scholarships are designed to empower current members by providing financial 
+                                assistance and recognizing their academic and professional achievements. Through the generosity of our 
+                                alumni, we continue to foster a culture of excellence and ambition, ensuring that every member has the 
+                                opportunity to succeed and make a lasting impact in the STEM fields.</p>
+                        </div>
+                    </div>
+                    <img src={alum_vector1} className="w-80 h-60 absolute transform translate-y-[-207%] z-0"/>
+                    <img src={alum_circles2} className="w-14 h-16 absolute transform translate-y-[-630%] translate-x-[270%] z-5"/>
+                    <img src={alum_shadow1} className="w-80 h-8 absolute transform translate-x-[100%] translate-y-[50%] z-0"/>
+                </div>
+                <div className="w-[35%] h-full relative">
+                    <img src={fep_award} className="w-[90%] h-[55%] mt-[24%] rounded-bl-[40px] rounded-tr-[40px] z-10 relative"/>
+                    <img src={alum_circles1} className="w-14 h-24 absolute transform translate-x-[728%] translate-y-[-75%] z-0"/>
+                </div>
+                <div className="flex flex-col flex-grow justify-start items-end">
+                    <img src={alum_triangle} className="w-28 h-28"/>
+                    <img src={alum_dots1} className="w-28 h-28"/>
                 </div>
             </div>
 
-            <div className="flex items-start max-w-[1270px] w-full h-[413px] mt-20">
-                <div className="mr-8 flex flex-col justify-center">
-                    <h2 className="text-black font-inter text-4xl font-medium" style={{ letterSpacing: '8px' }}>
-                        PROFESSIONAL NETWORK
-                    </h2>
-                    <p className="text-black font-inter text-base leading-7">
-                        Provide your contact information and a little bit about your career path to allow members interested in your career to reach out!
-                    </p>
+            {/* Network w pros */}
+            <div className="w-full h-[540px] bg-white flex flex-row relative">
+                <img src={alum_picnic} className="w-[35%] h-[55%] mt-[8%] ml-[16%] z-10"></img>
+                <div className="w-1/3 h-[65%] bg-[#f8f8f8] rounded-t-[60px] rounded-br-[60px] ml-[4%] mt-[7.5%] z-10">
```

Comment:
Note that this is what it looks like on bigger screens 

![image](https://github.com/user-attachments/assets/68f53bbb-d1fb-49bc-b269-b728198f4bb8)